tla-assignments/src/main/scala/at/forsyte/apalache/tla/assignments/Reordering.scala

```diff
@@ -0,0 +1,50 @@
+package at.forsyte.apalache.tla.assignments
+
+import at.forsyte.apalache.tla.lir.oper.TlaBoolOper
+import at.forsyte.apalache.tla.lir.{LetInEx, OperEx, TlaEx, TlaOperDecl}
+import at.forsyte.apalache.tla.lir.transformations.{TlaExTransformation, TransformationTracker}
+
+class Reordering[T]( ord : Ordering[T], rankingFn : TlaEx => T, tracker : TransformationTracker ) {
+
+  def reorder : TlaExTransformation = tracker.track {
+    // For now, only /\ and \/ are reordered, but the same process works for all commutative operators
```

Comment:
No plans atm, the only other control-flow relevant operator I can think of is `CASE`, which is commutative in base TLA+, but not actually commutative in Apalache. 
The opposite actually, it might happen that we end up not reordering `\/`. But for now, no TODO, this is complete as is.